src/doc/DocAnnotator.js

```diff
@@ -169,6 +169,11 @@ class DocAnnotator extends Annotator {
             ];
             let [x, y] = browserCoordinates;
 
+            // If click is outside the page, ignore
+            if (x < 0 || x > pageWidth || y < 0 || y > pageHeight) {
```

Comment:
Might be worth moving this into a reusable util method in docUtil.js and combine the check w/ the one below